onnx_ops/conftest.py

```diff
@@ -385,23 +400,37 @@ def __init__(
         except:
             outs = str(process.stdout)
 
-        super().__init__(
-            f"Error invoking iree-compile\n"
-            f"Error code: {process.returncode}\n"
-            f"Stderr diagnostics:\n{errs}\n\n"
-            f"Stdout diagnostics:\n{outs}\n\n"
-            f"Compiled with:\n"
-            f"  cd {cwd} && {compile_cmd}\n\n"
+        test_github_url = (
+            "https://github.com/iree-org/iree-test-suites/blob/main/onnx_ops/"
+            + cwd.relative_to(THIS_DIR).as_posix()
```

Comment:
This URL could also be derived from git or the environment somehow:
* not `blob/main`, but `blob/[CURRENT HASH]`
* not necessarily `iree-org/iree-test-suites`

Seems fine to just hardcode it for now. The test case source files don't change frequently.